requirements.txt

```diff
@@ -6,7 +6,7 @@
 # To install release requirements: inv requirements --release
 
 invoke==0.15.0
-Werkzeug==0.10.4  # pyup: ignore # newer versions break local development
```

Comment:
This pyup comment doesn't seem to be relevant though I couldn't find must info about what the problem actually was.